diff -uNr a/generate-db.js b/generate-db.js
--- a/generate-db.js
+++ b/generate-db.js
@@ -150,7 +150,7 @@
   var url;
   if (pkg_format === "debian")
     // Fetch the Contents.gz file for the given architecture from the apt mirror
-    url = `${repo.url}/dists/${repo.distribution}/Contents-${arch}.gz`;
+    url = `${repo.url}/dists/${repo.distribution}/${repo.component ?? "main"}/Contents-${arch}.gz`;
   else if (pkg_format === "pacman")
     // Fetch the ${repo}.files file for the given architecture from the pacman mirror
     url = `${repo.url}/${arch}/${repo.distribution}.files`;
